src/hooks/addEmbedded.js

```diff
@@ -89,9 +89,9 @@ module.exports = function(context) {
     myProj.parseSync();
     addRunpathSearchBuildProperty(myProj);
 
-    var projectName = myProj.getFirstTarget().firstTarget.name;
+    const projectName = myProj.getFirstTarget().firstTarget.name.replace(/^["]+|["]+$/g, '');
```

Comment:
`myProj.getFirstTarget().firstTarget` returns


![image](https://cloud.githubusercontent.com/assets/6878222/25094655/d398b86c-235d-11e7-9121-10569ef55260.png)

as you can see there's double quotes around the project name. so at the moment of resolving the full path, it ends up being incorrect such as

`"myProjectFolder"/plugins/rainforest-cordova-plugin`

correct (no quotes):

`myProjectFolder/plugins/rainforest-cordova-plugin`